distributions/aws/examples/rds-s3/README.md

```diff
@@ -59,8 +57,10 @@ eksctl create cluster \
 **Important :**  
 You must make sure you have an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for your cluster and that it was added from `eksctl` >= `0.56` or if you already have an OIDC provider in place, then you must make sure you have the tag `alpha.eksctl.io/cluster-name` with the cluster name as its value. If you don't have the tag, you can add it via the AWS Console by navigating to IAM->Identity providers->Your OIDC->Tags.
 
-### 2. Setup RDS and S3
-#### **Automated setup**
+## 2.0 Setup RDS, S3 and configure Secrets
```

Comment:
Really appreciate the changes here, much more readable.  
The only thing I'd say is check to make sure `cognito-rds-s3` README link is updated for this new H title name.  
Check the 1. Prerequisite par `RDS & S3 from [this guide](https://github.com/mbaijal/kubeflow-manifests/blob/fix-rds-readme/distributions/aws/examples/rds-s3/README.md#2-setup-rds-and-s3) to:`